odoo/addons/base/data/res_currency_rate_demo.xml

```diff
@@ -11,7 +11,7 @@
         <record id="rateUSDbis" model="res.currency.rate">
             <field name="rate">1.5289</field>
             <field name="currency_id" ref="USD"/>
-            <field eval="time.strftime('%Y-06-06')" name="name"/>
+            <field eval="time.strftime('%Y-01-01')" name="name"/>
```

Comment:
Was to prevent creating rate in the future. rateUSDbis removed now anyway

Reply:
It would maybe have been nice to create the rates with https://github.com/odoo/odoo/commit/d31c5c70e0411be38488e80112b656215060d628, this way you can create it for the right currencies depending on the company

Reply:
Sure a nice thing to do. But i think @tsb-odoo want's this task to go through and would prefer this be done in a new task.
Demo USD rates removed for now avoiding this weird 1 USD = 1.5289 USD thing.

Reply:
> Surely a nice thing to do. But i think @tsb-odoo want's this task to go through and would prefer this be done in a new task.
> Demo USD rates removed for now avoiding this weird 1 USD = 1.5289 USD thing.

@jbw-odoo Need anything from me on this ?

Reply:
If you install the database by specifying a country (from the UI), or if you change the company's currency you will have an issue like this one but with the other country. Is it ok @tsb-odoo ?

Reply:
The more I look at it the more it bothers me. I'll make the changes.